File: 1019-squares-of-a-sorted-array/1019-squares-of-a-sorted-array.java
class Solution {
    public int[] sortedSquares(int[] nums) {
        int[] ans = new int[nums.length]; 
        int i = 0; 

        for( int num : nums){
            ans[i++] = Math.abs( num*num); 
        }
        Arrays.sort(ans); 
        return ans;
        
    }
}

File: 1019-squares-of-a-sorted-array/1019-squares-of-a-sorted-array.cpp
class Solution {
public:
    vector<int> sortedSquares(vector<int>& nums) {
        vector<int>ans(nums.size()); 
        int i= 0;

        for( auto num : nums){
            ans[i++] = abs(num*num); 
        }
        sort(ans.begin(),ans.end()); 
        return ans;
        
    }
};